recipe/0001-use-sys-deps-add-libs-and-make-install-osxarm64.patch

```diff
@@ -0,0 +1,55 @@
+From 79f522daa355958adaebb6d4dea14b8c00219119 Mon Sep 17 00:00:00 2001
+From: ngam <67342040+ngam@users.noreply.github.com>
+Date: Sun, 30 Jan 2022 22:42:44 -0500
+Subject: [PATCH 1/3] use sys deps, add libs, and make install
+
+---
+ setup.py | 18 ++++++++++++++++--
+ 1 file changed, 16 insertions(+), 2 deletions(-)
+
+diff --git a/setup.py b/setup.py
+index 8604d465e..ecbb31604 100644
+--- a/setup.py
++++ b/setup.py
+@@ -125,6 +125,19 @@ class BuildPyLibVWBindingsModule(_build_ext):
+             )
+             cmake_args.append("-DPython_INCLUDE_DIR={}".format(get_python_inc()))
+ 
++            cmake_args += [
```

Comment:
The package supports a global option to supply cmake args. Maybe helpful here? https://github.com/VowpalWabbit/vowpal_wabbit/blob/9471e85c80c93cc40ad2d8ec7788f9d5d7c1ee17/setup.py#L30

Reply:
Yes, these patches were done this way because I was simultaneously building locally for the upstream version as well as trying to make it work for this PR. Once we settle on a path forward, I will rebase the whole thing and use the cmake-options since it will be significantly cleaner!

Reply:
Sounds good!